frontend/src/pages/Loginpage.js

```diff
@@ -8,43 +11,56 @@ const initialState = {
 
 export default function Loginpage() {
 
-const [credentials, setCrededtials] = useState(initialState);
-const {login} = useContext(AuthContext)
+    const [credentials, setCrededtials] = useState(initialState);
+    const {login} = useContext(AuthContext)
 
+    const handleChange = event => {
+        setCrededtials({...credentials, [event.target.name]: event.target.value})
+    }
 
-const handleChange = event => {
```

Comment:
Very smart way to handle these things - but for the future you need a check if the credentials are correct and User Feedback :-)

Reply:
fair point, taken :-) thanks. noted for further dev steps.